Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.h

```diff
@@ -36,6 +36,14 @@ namespace AzToolsFramework
         void AddAction(int sortKey, AZStd::string actionIdentifier);
         void AddSubMenu(int sortKey, AZStd::string menuIdentifier);
         void AddWidget(int sortKey, QWidget* widget);
+
+        // Returns whether the action or menu queried is contained in this menu.
+        bool ContainsAction(const AZStd::string& actionIdentifier) const;
+        bool ContainsSubMenu(const AZStd::string& menuIdentifier) const;
+
+        // Returns the sort key for the queried action or menu, or 0 if it's not found.
+        int GetActionSortKey(const AZStd::string& actionIdentifier) const;
+        int GetSubMenuSortKey(const AZStd::string& menuIdentifier) const;
```

Comment:
suggestion: If we do want to still use an int and not an optional, a more obvious sentinel value such as -1 might be better (I'd still personally lean towards an optional to make things more explicit but it's totally up to you)